.travis.yml

```diff
@@ -1,23 +1,27 @@
+sudo: false
 language: python
-sudo: require
-python: 
-   - "2.7"
+python:
+  - 2.7
+
 addons:
-   apt:
-      packages:
-         - libxml2-dev
-         - libxslt-dev
-         - libsofthsm
-         - softhsm
-         - opensc
-         - libengine-pkcs11-openssl
-before_install: 
-   - sudo apt-get -qq update 
-   - sudo apt-get -y install libxml2-dev libxslt-dev libsofthsm softhsm opensc libengine-pkcs11-openssl
-   - pip install .
-   - pip install nose
-   - sh scripts/install-pykcs11.sh
-   - pip install coveralls
-script: "coverage run --source=xmlsec setup.py nosetests"
+  apt:
+    packages:
```

Comment:
I my virgin Ubuntu trusty I also had to install libssl-dev and libffi-dev in order to get a complete `pip install coveralls` without warnings. Don't know if this might be installed by default in the travis docker or not?!